src/htmldocvisitor.cpp

```diff
@@ -401,8 +401,7 @@ void HtmlDocVisitor::operator()(const DocURL &u)
   else // web address
   {
     m_t << "<a href=\"";
-    filter(u.url());
-    m_t << "\">";
+    m_t << u.url() << "\">";
```

Comment:
Looks to me an incorrect change by accident / not merged properly with master. The filter was explicitly added for #11404 to be reverted.